.readme-partials.yaml

```diff
@@ -0,0 +1,101 @@
+introduction: |-
+  > This module provides Cloud Debugger support for Node.js applications.
+  Cloud Debugger is a feature of Google Cloud Platform that lets you debug your
+  applications in production without stopping or pausing your application.
+
+body: |-
+  ## Debugger Agent Settings
+
+  You can customize the behaviour of the automatic debugger agent. See
```

Comment:
Generally avoid statements  that start with "You can..." re-word as "To customize the behavior of the automatic debugger agent see..."